assembl/static2/js/app/actions/debateActions.js

```diff
@@ -33,7 +33,7 @@ export const fetchDebateData = (debateId) => {
       dispatch(resolvedFetchDebateData(debateData));
     }).catch((error) => {
       const firstError = error[0];
-      if (firstError.status === 401) {
+      if (firstError && 'status' in firstError && firstError.status === 401) {
```

Comment:
I think, based on the contract we have jsonError in backend, this `error` object will always exist. Otherwise this block of code will not be hit. But anyways, it's fine to keep it, I think.